app/auth/controller/loginController.js

```diff
@@ -1,47 +1,47 @@
-angular.module('openhds')
-    .controller('LoginController',
-                ['$rootScope', '$location', '$http', LoginController]);
+angular.module('LoginModule', [])
 
-function LoginController($rootScope, $location, $http) {
-    var vm = this;
-    var configUrl = "/config.json";
-    var restUrlConfigKey = 'openhdsRest';
-    var homeUrl = "/baseline";
+    .controller('LoginController',
+        ['$rootScope', '$location', '$http',
+            function($rootScope, $location, $http) {
```

Comment:
I think the preferred style is to used named functions for components 

https://github.com/johnpapa/angular-styleguide/tree/master/a1#named-vs-anonymous-functions